demos/_unit-test/scope-builder.php

```diff
@@ -15,14 +19,15 @@
 $model = new Stat($app->db, ['caption' => 'Demo Stat']);
 
 $project = new Condition('project_name', Condition::OPERATOR_REGEXP, '[a-zA-Z]');
-$brazil = new Condition('client_country_iso', '=', 'Brazil');
+$brazil = new Condition('client_country_iso', '=', '30');
```

Comment:
use ISO code (BR) lookup preferably

Reply:
@mvorisek is right - iso field should contain iso code not id.

Reply:
This was per the original implementation from @georgehristov - I can check how/why it does not store ISO instead.
@georgehristov - any input on this one. 
Besides there was also a bug before as it was using the name instead of id value when using $field->reference

Reply:
Yes using name there also looks wrong. It should be value of that field so in this case iso value. If you lookup by client_country_id, then country_id value etc.

Reply:
done. Now use their_field to properly create dropdowns list key-value pair. Therefore will use 'BR' in query value.